client/webpack.config.js

```diff
@@ -275,8 +273,9 @@ const webpackConfig = {
 			__i18n_text_domain__: JSON.stringify( 'default' ),
 			global: 'window',
 		} ),
+		// Node polyfills
```

Comment:
Not related to this PR?

Reply:
Correct, this was solved in #55971, but I added it here to get the tests working. #55971 is only needed because of dependencies this update adds.